Description: Fix privacy-breach-generic by removing support for ancient
 browsers.
Author: Benjamin Drung <benjamin.drung@profitbricks.com>

--- a/flower/templates/base.html
+++ b/flower/templates/base.html
@@ -21,10 +21,6 @@
     {% block extra_styles %}
     {% end %}
 
-    <!--[if lt IE 9]>
-      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
-
     <!-- Le fav and touch icons -->
     <link rel="shortcut icon" href="{{ static_url('favicon.ico') }}">
     <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ static_url('img/apple-touch-icon-144-precomposed.png') }}">
